specification/network/resource-manager/Microsoft.Network/stable/2018-10-01/publicIpAddress.json

```diff
@@ -452,6 +456,37 @@
       },
       "description": "Contains FQDN of the DNS record associated with the public IP address"
     },
+    "DdoSSettings": {
+      "properties": {
+        "ddosCustomPolicy": {
+          "readOnly": false,
+          "type": "string",          
+          "description": "The DDoS custom policy associated with the public IP."
+        },
+        "protectionCoverage": {
+          "readOnly": false,
+          "type": "string",
+          "enum": [
+            "Basic",
+            "Standard"            
+          ],
+          "description": "The type of the DDoS protection plan associated with the public IP."
+        },
+        "domainNameLabel": {
```

Comment:
Why is domainNameLabel, fqdn etc in the DDOS settings?

Reply:
removed